### PR TITLE
Travis and Hspec set up

### PR DESCRIPTION
**Pull request by ibarrae**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
 
 packages:
-- .
\ No newline at end of file
+- .
+
+ghc-options:
+  "$locals": -Wall
+  "$targets": -Werror
+  "$everything": -O2
```
Comment (CristhianMotoche) [inline stack.yaml:9]: @kinorius123 Are these options added to the cabal file too?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: haskell
+
+ghc:
+  - "8.2"
```
Comment (dbalseiro) [inline .travis.yml]: I think LTS 12.1 uses GHC 8.4

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  - "8.0"
+  - "7.10"
```
Comment (dbalseiro) [inline .travis.yml]: Not sure if we want to test backwards compatibility here... I think we better stick to one version...

**Review by dbalseiro — CHANGES_REQUESTED** [review]
Just a couple of comments regarding the GHC version

**Review by CristhianMotoche — APPROVED** [review]

**Review by dbalseiro — APPROVED** [review]

**Pull request merged**